src/lib.rs

```diff
@@ -318,6 +318,33 @@ where
     }
 }
 
+impl<T, N> Counter<T, N>
+where
+    T: Hash + Eq,
+{
+    /// Returns the sum of the counts.
+    ///
+    /// Use [`len`] to get the number of elements in the counter and use `total` to get the sum of
+    /// their counts.
+    ///
+    /// [`len`]: struct.Counter.html#method.len
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// # use counter::Counter;
+    /// let counter = Counter::init("abracadabra".chars());
+    /// assert_eq!(counter.total::<usize>(), 11);
+    /// assert_eq!(counter.len(), 5);
+    /// ```
+    pub fn total<'a, S>(&'a self) -> S
+    where
+        S: Sum<&'a N>,
```

Comment:
```suggestion
    where
        S: iter::Sum<&'a N>,
```

As this bound is used only here, I'd prefer not to `use` it throughout the file.